beesdoo_shift/wizard/subscribe.xml

```diff
@@ -16,12 +16,16 @@
                     <field name="working_mode" />
                     <field name="exempt_reason_id" attrs="{'invisible':[('working_mode', '!=', 'exempt')]}"/>
                     <field name="shift_id" domain="[('remaining_worker', '>', 0)]" attrs="{'invisible': [('working_mode', '!=', 'regular')]}"/>
+                    <field name="nb_shifts" attrs="{'invisible': True}"/>
                     <field name="irregular_start_date" attrs="{'invisible': [('working_mode', '!=', 'irregular')]}" />
                     <field name="super" />
                     <field name="reset_counter" />
                     <field name="reset_compensation_counter" /> <!-- TODO access right -->
                 </group>
                 <footer>
+                    <div class="alert alert-danger" role="alert" attrs="{'invisible': [('nb_shifts', '&lt;', 2)]}">
+                      Current worker has more than one shift, subscribing him to a new shift will erase all previous shifts.
```

Comment:
@tfrancoi @remytms Will this text be picked up by gettext (for translation)? Or is there a tag I should use? Another method?

Reply:
@aborsu Normally, it will be caught by the translation system. To be sure, after your module is updated, you can go in Configuration > Traductions > Termes de l'application > Termes traduits, there you can find it.